### ISSUE TITLE
Build script keeps on looping

### ISSUE THREAD

**Issue by Guyda** [antonybudianto/cra-universal#93]
Hi,
I've been having a weird issue with `npx cra-universal build` where it just loops and displays the following information:

>  crau >  NODE_ENV is "production"
>  crau >  "server" folder found on CRA client, using this one
>  crau >  "./server/.babelrc" found, using this one.
>  crau >  Build clean done.
>  crau >  Building CRA client...

Nothing is being built nor compiled, just looping. I'm using redux, react-router following the information in the wiki. I had to add 2 plugins for the server to work and recognize es6 arrow functions and object spreads:
`"plugins": ["transform-object-rest-spread", "transform-class-properties"]`

Not sure what I'm missing in my setup
Thanks

**Comment by antonybudianto** [comment]
Hi.

Can you show me your package json? Probably the "build" script is modified, it should be "react-scripts build"

**Comment by Guyda** [comment]
I'm an idiot, I had to eject a while back and I forgot I renamed the original "build" script to "build-js"... I changed the name back and it built it all in the ./dist folder. No more looping. 
What my scripts were:

> "scripts": {
>   ...
>   "build-js": "node scripts/build.js",
>   "build": "npx cra-universal build"
> }

What I changed them to:

> "scripts": {
>   ...
>   "build": "node scripts/build.js",
>   "crau:build": "npx cra-universal build"
> }

My bad if I raised unnecessary concerns, I've been going in circles for 2 days on this and couldn't see what was wrong with my set up. Thanks a lot for the quick response and most importantly for the work you do.
